vendor/BUILD

```diff
@@ -17120,3 +17110,13 @@ go_library(
         "//vendor:k8s.io/kube-aggregator/pkg/apis/apiregistration/install",
     ],
 )
+
+go_library(
+    name = "github.com/ClusterHQ/flocker-go",
```

Comment:
I ran update-bazel.sh from my mac and these were the changes. 
Do you want me to delete these ?